Formula/python@3.11.rb

```diff
@@ -419,22 +420,24 @@ def sitecustomize
           sys.path.extend(library_packages)
           # the Cellar site-packages is a symlink to the HOMEBREW_PREFIX
           # site_packages; prefer the shorter paths
-          long_prefix = re.compile(r'#{rack}/[0-9._abrc]+/Frameworks/Python.framework/Versions/#{version.major_minor}/lib/python#{version.major_minor}/site-packages')
+          long_prefix = re.compile(r'#{rack}/[0-9\\._abrc]+/Frameworks/Python\\.framework/Versions/#{version.major_minor}/lib/python#{version.major_minor}/site-packages')
```

Comment:
Actually, the first escape probably isn't needed as it is inside character class so `.` is a literal. The second is for `Python.framework` so better to escape.

I don't think it causes any issues as is, so might just handle any other sitecustomize changes in separate PR along with formatting changes and maybe fixing logic for Linux.

Reply:
yeah first probably isn't - this diff looks weirder as it was original based before the commit that removed all (broken) escapes from the formula. There was originally one backslash for both.